Week3/homework/step3.js

```diff
@@ -2,7 +2,10 @@
 
 function createBase(base) {
   // Replace this comment and the next line with your code
-  console.log(base);
+  function add(number) {
+    return base + number;
+  }
+  return add;
 }
 
 const addSix = createBase(6);
```

Comment:
You can also directly return an anonymous function, e.g.:

```js
function createBase(base) {
  return function(number) {
    return base + number;
  };
}
```

Or, if you are adventurous, use arrow functions:

```js
const createBase = base => number => base + number;
```